Exercise_7/scripts/ticTacToe/Painter.js

```diff
@@ -0,0 +1,17 @@
+const Painter = () => {
+    const boardFieldClass = 'board__field';
+    const boardFieldHighlightedClass = 'board__field_highlighted'
+    return {
+        drawSign: (index, sign) => document.getElementsByClassName(boardFieldClass)[index].innerText = sign,
+
+        highlightField: index => {
+            document.getElementsByClassName(boardFieldHighlightedClass)[0].classList.remove(boardFieldHighlightedClass);
+            document.getElementsByClassName(boardFieldClass)[index].className += " " + boardFieldHighlightedClass;
```

Comment:
Please, replace the right part to the string template.
https://github.com/airbnb/javascript/blob/master/README.md#es6-template-literals